Resources/Prototypes/Entities/Objects/Specific/Janitorial/janitor.yml

```
@@ -1,337 +1,337 @@
+    enabled: false
+  - type: Physics
+    bodyType: Dynamic
+  - type: Fixtures
+    fixtures:
+      slips:
+        shape:
+          !type:PhysShapeAabb
+          bounds: "-0.2,-0.2,0.2,0.2"
+        hard: false
+        layer:
+          - LowImpassable
```

yamllint

[warning] wrong indentation: expected 8 but found 10

(indentation)